app/components/support_interface/interview_details_component.rb

```diff
@@ -0,0 +1,38 @@
+module SupportInterface
+  class InterviewDetailsComponent < ViewComponent::Base
+    include ViewHelper
+
+    attr_reader :interview
+
+    def initialize(interview:)
+      @interview = interview
+    end
+
+    def rows
+      [
+        {
+          key: 'Provider',
+          value: interview.provider.name,
+        },
+        {
+          key: 'Meeting details',
+          value: interview.location,
+        },
+        {
+          key: 'Created at',
+          value: interview.created_at.to_s(:govuk_date_and_time),
+        },
+        {
+          key: 'Updated at',
+          value: interview.created_at.to_s(:govuk_date_and_time),
+        },
+      ]
+    end
+
+  private
+
+    def interview_status
+      interview.cancelled_at.present? ? 'Cancelled' : 'Active'
```

Comment:
```suggestion
      interview.discarded? ? 'Cancelled' : 'Active'
```
We should use the [discard gem's API](https://github.com/jhawthorn/discard#discard-a-record) to check if an interview has been cancelled or not. Maybe we can alias the method on the interview model for clarity.

Reply:
Ok, I'll add a `cancelled?` alias to the model.